src/DDTrace/OpenTracer/Tracer.php

```diff
@@ -58,7 +58,10 @@ private function deconstructStartSpanOptions(\OpenTracing\StartSpanOptions $obj)
         }
 
         $options['finish_span_on_close'] = $obj->shouldFinishSpanOnClose();
-        $options['ignore_active_span'] = $obj->shouldIgnoreActiveSpan();
+        if (\method_exists($obj, 'shouldIgnoreActiveSpan')) {
```

Comment:
@SammyK while reviewing this PR please  pay attention to this. I added this so we are compatible with all versions also in tests. Note that were compatible with 5.6 before, unless the user used an instance of `\OpenTracing\StartSpanOptions` as `$options` that I'd say is the common approach if they just use open tracing to add so spans.

Reply:
Thanks for the investigation, fix, and comment @labbati. 👍